bindings/python/src/OpenSpaceToolkitPhysicsPy/Environment/Atmospheric/Earth/Weather.cpp

```diff
@@ -0,0 +1,17 @@
+/// Apache License 2.0
+
+#include <OpenSpaceToolkitPhysicsPy/Environment/Atmospheric/Earth/Weather/CSSISpaceWeather.cpp>
+#include <OpenSpaceToolkitPhysicsPy/Environment/Atmospheric/Earth/Weather/Manager.cpp>
+
+inline void OpenSpaceToolkitPhysicsPy_Environment_Atmospheric_Earth_Weather(pybind11::module& aModule)
```

Comment:
Nit:
```suggestion
inline void OpenSpaceToolkitPhysicsPy_Environment_Atmospheric_Earth_SpaceWeather(pybind11::module& aModule)
```
Maybe weather is a bit too generic?
If you agree then we can update the package path to `space_weather`

Reply:
then we need to keep it in sync with cpp code

Reply:
I think the difference between `Weather` and `SpaceWeather` is pretty minimal in the context of OSTk.

Now that I'm looking at it though, I think we can completely remove this subdirectory. It's already under `Atmospheric`, so weather is a bit redundant. What's more is that it deviates from what we do for some of the other things with Managers.

Gonna just nix this directory. Thanks for the nit :)